search-index/text2embeddings.py

```diff
@@ -323,6 +367,22 @@ def run_cli(
 
     logger.info(f"Saved embeddings and IDs for text and descriptions to {output_dir}")
 
+    if s3:
+        logger.info(f"Writing embeddings to S3 bucket {output_dir}")
+        output_dir.upload_from(embs_output_path, force_overwrite_to_cloud=True)
```

Comment:
What's the reason for setting `force_overwrite_to_cloud=True`? Timestamping files should ensure their uniqueness so it could be quite useful to find out if a file is going to be overwritten, as this would mean timestamping is probably broken.

Reply:
Nice spot. I think this is because I I was debugging with a subset of 1 file for speed but the time string is only 1m accuracy, so I ended up running the script twice within a minute and wrote this in to avoid the bugs, copy and pasted it and forgot to remove! Fixed.